ibm_cloud_sdk_core/__init__.py

```diff
@@ -17,3 +17,4 @@
 from .detailed_response import DetailedResponse
 from .iam_token_manager import IAMTokenManager
 from .api_exception import ApiException
+from .utils import datetime_to_string, string_to_datetime
```

Comment:
Does this mean that a generated .py file no longer needs to explicitly import these functions as long as it imports the "ibm_cloud_sdk_core" module?

Reply:
The import would look something like:
`from ibm_cloud_sdk_core import string_to_datetime`

or if you want to import the entire module
`
import ibm_cloud_sdk_core

and the using the module, call the function like this:
ibm_cloud_sdk_core.string_to_datetime()
`